packages/fxa-settings/src/components/Settings/ProgressBar/en.ftl

```diff
@@ -4,4 +4,4 @@
 # Variables:
 #   $currentStep (number) - the step which the user is currently on
 #   $numberOfSteps (number) - the total number of steps in a given flow
-progress-bar-aria-label = A progress bar showing that the user is on step { $currentStep } of { $numberOfSteps }.
+progress-bar-aria-label = Step { $currentStep } of { $numberOfSteps }.
```

Comment:
This will need a new string ID.

Reply:
Ah yes, thank you! I'd updated the id in the code but missed it here. Should be good now!